GOVERNANCE.md

```diff
@@ -0,0 +1,153 @@
+# Eiffel Community Governance
+
+Eiffel Community initiated the work to establish governance and put the necessary structures in place in order to have a healthy, sustainable,
+and attractive community.
+
+The community aims to adapt and follow the practices employed by other successful open source communities with minor adjustments based on
+Eiffel Community feedback.
+
+This document contains information regarding the proposed governance model to be employed by the Eiffel Community and is subject to change.
+First official version of the document will be available after the reviews of the Technical Committee (TC) and Eiffel Community at large.
+
+# Bootstrap Technical Committee
+
+During the bootstrap phase, Eiffel Community will be served by a Bootstrap TC to ensure necessary processes and procedures are in place to help transition to a fully elected TC.
+
+Bootstrap TC is responsible for governance of the Eiffel Open Source Project during the bootstrap phase and until the transition to a fully elected TC is concluded.
+
+## TC Members
+
+Members of the Bootstrap TC are core stakeholders and/or contributors.
+
+The committee will consist of **X** voting members.
+
+Members are the individuals listed below:
+
+* TBD
+
+After the bootstrap phase, the size, makeup and the procedure for determining the members of the committee will be as determined by the committee and documented within committee Procedures.
+
+The meetings of the committee are intended to be open to the public and can be conducted electronically, via teleconference, or in person.
+
+The members of the committee MUST
+
+* Represent a cross-section of interests, not just one company
+* Balance technical, architectural, and governance expertise since its initial mission is the establishment of structure around contributions, community, and decision-making
+* Hold staggered terms, sufficient to ensure an orderly transition of power via elections as designed and implemented by the committee (see below for specific deliverables)
+* Provide designated alternates in cases where quorum is required but not attainable with the current set of members
+
+## TC Chair
+
+The committee elects a TC Chair from within members, who will preside over meetings of the committee and will serve until their resignation or replacement by the committee.
+The term of the chair starts once the chair election is concluded and ends in conjunction with the term of the committee.
```

Comment:
As the TC chair term is open ended, this sentence is not necessary.  Suggest removing and focusing on the role and expectations.

Reply:
will amend